terraform/aws/components/gp2gp/gpc-consumer-variables.tf

```diff
@@ -189,4 +201,10 @@ variable secret_name_spine_sub_ca_cert {
 variable secret_name_sds_apikey {
   type = string
   default = "opentest-client-key"
-}
\ No newline at end of file
+}
+
+variable "gpc-consumer_empty_string" {
```

Comment:
What is this variable? It is actually needed?

Reply:
I defined this var for testing purposes to populate empty string to gpcc secret certs, but data secrets gave error on passing empty strings. It is removed now.